torch/_ops.py

```diff
@@ -242,14 +242,24 @@ def dispatch(self, dispatch_key, *args, **kwargs):
         return kernel(*args, **kwargs)
 
     def __call__(self, *args, **kwargs):
-        flat_args = _to_flat_tuple(args, kwargs)
-        if torch.overrides.has_torch_function(flat_args):
-            return torch.overrides.handle_torch_function(
-                self, flat_args, *args, **kwargs
+        # Dynamo already traces the body of HigherOrderOp beforehand when it
+        # so no need to trace into it.
+        from torch._dynamo.eval_frame import disable
```

Comment:
This assumes that `torch._dynamo` is already imported, which it might not be. We need to add an `import torch._dynamo` before this line